modules/test/playwright/tests/journal-web/main/journalPage.spec.ts

```diff
@@ -379,3 +379,65 @@ test(
 		expect(/[[{]/.test(inputValue || '')).toBeFalsy();
 	}
 );
+
+test(
+	'Validate Folder Count After Creating Folders and Filtering by Approved',
```

Comment:
I would rename to something like `Folders Comes First when Having Multiple Pages and Filtering by Approved`

I think it better explains the use case.